README.md

```diff
@@ -14,6 +14,9 @@
   <a href='https://app.fossa.com/projects/custom%2B4458%2Fgithub.com%2Fkeplergl%2Fkepler.gl?ref=badge_shield'>
     <img src='https://app.fossa.com/api/projects/custom%2B4458%2Fgithub.com%2Fkeplergl%2Fkepler.gl.svg?type=shield' alt='Fossa' />
   </a>
+  <a href="https://app.netlify.com/sites/keplergl/deploys" alt="Netlify Status">
+    <img src="https://api.netlify.com/api/v1/badges/0c9b895c-acd0-43fd-8af7-fe960181b686/deploy-status" />
```

Comment:
You can use `shield.io` to build netlify badge consistent with our other badge style. replace ths image.src with
`https://img.shields.io/endpoint.svg?url=https%3A%2F%2Fdeveloper.oswaldlabs.com%2Fnetlify-status%2F0c9b895c-acd0-43fd-8af7-fe960181b686?style=flat`

More detail in this thread:
https://github.com/badges/shields/issues/3129#issuecomment-470032754

Reply:
Done